script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -397,7 +399,7 @@ Even through the microphone, I can sense Mario's disappointment. His despair. Hi
 [sha:b874a8bc61b39a4314e9a0ea1cb29d14fcc4ae9e]{
 -- Page 102, Offset 30829.
 -- 「そう言った。アレらは私が開発した魔術基盤。
-"Indeed. They are things I constructed to underpin my Magecraft.
+"Indeed. They are %{i}magecraft foundations%{/i}, that I have constructed.
```

Comment:
no caps for compound term? Also no comma I think.

Reply:
caps them